webpack/scenes/ModuleStreams/Details/ModuleStreamDetailInfo.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Table } from 'react-bootstrap';
+
+const displayMap = {
+  name: 'Name',
```

Comment:
I can't say for sure, but I don't think it matters since its rendered into html. Someone who is more familiar with how our translations work can maybe answer this better